operations/helm/charts/mimir-distributed/values.yaml

```
@@ -1103,6 +1227,48 @@ store_gateway:
   env: []
   extraEnvFrom: []
 
+  # -- Options to configure zone aware replication for store-gateway
+  zoneAwareReplication:
+    # -- Enable zone aware replication for store-gateway
+    enabled: true
+    # -- Maximum number of store-gateways that can be unavailable per zone during rollout
+    maxUnavailable: 10
+    # -- topologyKey to use in pod anti-affinity. If unset, no anti-affinity rules are generated. If set, the generated anti-affinity rule makes sure that pods from different zones do not mix.
```

nit, can reject if you think this is too much explanaiton
```suggestion
    # -- topologyKey to use in pod anti-affinity. If unset, no anti-affinity rules are generated. If set, the generated anti-affinity rule makes sure that pods from different zones do not mix on nodes with the same topology key value.
```